packages/compass-collection/src/components/mock-data-generator-modal/script-screen.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import {
+  Body,
+  Code,
+  Copyable,
+  css,
+  cx,
+  InlineCode,
+  Language,
+  Link,
+  Overline,
+  palette,
+  spacing,
+  useDarkMode,
+} from '@mongodb-js/compass-components';
+import { useConnectionInfo } from '@mongodb-js/compass-connections/provider';
+
+const RUN_SCRIPT_COMMAND = `
+mongosh "mongodb+srv://<your-cluster>.mongodb.net/<your-database>" \\
+  --username <your-username> \\
+  --password "<your-password>" \\
+  mockdatascript.js
+`;
+
+const outerSectionStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[400],
+});
+
+const listStyles = css({
+  listStylePosition: 'inside',
+  listStyleType: 'disc',
+  marginLeft: spacing[200],
+});
+
+const copyableStyles = css({
+  marginLeft: spacing[400],
+});
+
+const sectionInstructionStyles = css({
+  margin: `${spacing[200]}px 0`,
+});
+
+const resourceSectionStyles = css({
+  padding: `${spacing[400]}px ${spacing[800]}px`,
+  borderRadius: spacing[400],
+});
+
+const resourceSectionLightStyles = css({
+  backgroundColor: palette.gray.light3,
+});
+
+const resourceSectionDarkStyles = css({
+  backgroundColor: palette.gray.dark3,
+});
+
+const resourceSectionHeader = css({
+  marginBottom: spacing[300],
+});
+
+const ScriptScreen = () => {
+  const isDarkMode = useDarkMode();
+  const connectionInfo = useConnectionInfo();
+
+  return (
+    <section className={outerSectionStyles}>
+      <section>
+        <Body as="h2" baseFontSize={16} weight="medium">
+          Prerequisites
+        </Body>
+        <Body className={css({ marginTop: spacing[200] })}>
```

Comment:
Don't use the `css` in render method, this creates a new class name every time component updates and is not performant, static styles can be defined with css outside of render, dynamic should be passed through the style prop.
It might be something a new lint rule could catch 🤔